dev-python/pycuda/pycuda-2018.1.1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
```

Comment:
Could you try with python3.7?

Reply:
I currently don't have a machine that uses python3.7, but upstream added several fixes for python3.7 in 2018.1.1 so I guess it should work.

Reply:
turns out it needs python3.7 enabled for boost which is not available yet.